tools/build.hpp

```diff
@@ -97,7 +101,7 @@ namespace antler {
 
       CLI::App*   subcommand = nullptr;
       std::string path;
-      uint32_t    jobs = 1;
+      uint32_t    jobs = 0;

```

Comment:
If we switch this to a negative number then we don't NEED to pass it to cmake. I generally dislike when we lack an option to leave things alone.
```suggestion
      int32_t    jobs = -1;
```

Reply:
And line 43 could switch to 
```cpp
if(jobs < 0)
   return system::execute("cmake", {"--build", bin_dir.string()});
return system::execute("cmake", {"--build", bin_dir.string(), "-j", std::to_string(jobs)});
```